Makefile

```diff
@@ -160,14 +160,67 @@ module-experimental-eirinifs:
 module-experimental-eirini_release:
 	$(MAKE) -C modules/experimental eirini_release
 
-# scf-only targets:
-.PHONY: scf-deploy
-scf-deploy:
-	@echo 'WARNING: target deprecated. Please use `make k8s/kubeconfig` and then `make scf` instead.'
-	@echo 'Kindly waiting for 20s…'; sleep 20
-	$(MAKE) clean buildir
-	$(MAKE) k8s kubeconfig scf
+# kubecf-only targets:
+.PHONY: kubecf-build
+kubecf-build: ##@kubecf Build chart from source and install KubeCF
+	$(MAKE) -C modules/kubecf build-from-source
+	$(MAKE) kubecf-gen-config
+	$(MAKE) -C modules/kubecf install
+
+.PHONY: kubecf-clean
+kubecf-clean: ##@kubecf Only delete installation of KubeCF & related files
+	$(MAKE) -C modules/kubecf clean
+
+.PHONY: sc	f
+kubecf: ##@STATES Delete if exists then deploy KubeCF in cluster
+	$(MAKE) -C modules/kubecf
+
+.PHONY: kubecf-chart
+kubecf-chart: ##@kubecf Only obtain KubeCF chart, by file or download
+	$(MAKE) -C modules/kubecf chart
+
+.PHONY: kubecf-gen-config
+kubecf-gen-config: ##@kubecf Only generate KubeCF config yamls
+	$(MAKE) -C modules/kubecf gen-config
+
+.PHONY: kubecf-install
+kubecf-install: ##@kubecf Only install KubeCF
+	$(MAKE) -C modules/kubecf install
 
+.PHONY: kubecf-upgrade
+kubecf-upgrade: ##@kubecf Only upgrade KubeCF
+	$(MAKE) -C modules/kubecf upgrade
+
+.PHONY: kubecf-login
+kubecf-login: ##@kubecf Only perform cf login against deployed KubeCF
+	$(MAKE) -C modules/kubecf login
+
+.PHONY: kubecf-minibroker
+kubecf-minibroker: ##@kubecf Deploy minibroker & services on KubeCF
+	$(MAKE) -C modules/kubecf minibroker
+
+.PHONY: kubecf-purge
+kubecf-purge: ##@kubecf Purge all apps, buildpacks and services from KubeCF
+	$(MAKE) -C modules/kubecf purge
+
+.PHONY: kubecf-build-stemcell
+kubecf-build-stemcell: ##@kubecf Build stemcell for KubeCF
+	$(MAKE) -C modules/kubecf stemcell_build
+
+# scf-only targets:
+ifeq "$(SCF_OPERATOR)" "true"
```

Comment:
Maybe enhance the comment a bit, so when we will look at this after few weeks, we will remember why we did this :slightly_smiling_face:

Reply:
True! many thanks, fixed.